doc/rtd/topics/faq.rst

```diff
@@ -151,6 +151,36 @@ provided to the system:
 As launching instances in the cloud can cost money and take a bit longer,
 sometimes it is easier to launch instances locally using Multipass or LXD:
 
+Why did cloud-init never complete?
+==================================
+
+To check if cloud-init is running still, run:
+```
+cloud-init status
+```
+
+To wait for clous-init to complete, run:
+```
+cloud-init status --wait
+```
+
+There are a number of reasons that cloud-init might "never complete. This list
+is not exhaustive, but attempts to enumerate potential causes:
+
+External reasons:
+-----------------
+- failed dependant services in the boot
+- bugs in the kernel or drivers
+- bugs in external userspace tools that are called by cloud-init
+
+## Internal reasons:
+--------------------
+- a command in `bootcmd` or `runcmd` that never completes (ex: running
+  `cloud-init status --wait` will wait forever on itself and never complete)
+- nonstandard configurations that disable timeouts or set extremely high
+  values ("never" is used in a loose sense here)
+- cloud-init is not immune to bugs
```

Comment:
I don't think this line adds much. I think everybody knows any software can have bugs.